server/db/migrate/20190320134223_remove_rating_from_restaurant.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveRatingFromRestaurant < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :restaurants, :rating, :integer
```

Comment:
The type isnt really needed when you clear it out :)

TIL: adding `type` and `options` helps create a rollback for this migration, so i guess its cool to keep it.
https://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/remove_column